pkg/gadgets/trace/dns/tracer/tracer.go

```diff
@@ -226,7 +243,7 @@ func parseDNSEvent(rawSample []byte) (*types.Event, error) {
 	}
 
 	// Convert name into a string with dots
-	event.DNSName = parseLabelSequence(rawSample[unsafe.Offsetof(bpfEvent.Name):])
+	event.DNSName = parseLabelSequence(bpfEvent.Name[:])
```

Comment:
I think this is equivalent -- is there a reason to prefer reading the data from `rawSample`?

Reply:
I suppose it's a leftover from previous iterations. It should be fine to change it.